src/CLI/actioner/sonic-cli-nat.py

```diff
@@ -0,0 +1,188 @@
+        path = cc.Path('/restconf/data/openconfig-nat:nat/instances/instance={id}/nat-pool', id=args[0])
+        return api.delete(path)
+
+    # Config NAT Binding
+    elif func == 'patch_openconfig_nat_nat_instances_instance_nat_acl_pool_binding_nat_acl_pool_binding_entry_config':
+        path = cc.Path('/restconf/data/openconfig-nat:nat/instances/instance={id}/nat-acl-pool-binding/nat-acl-pool-binding-entry={name}/config', id=args[0],name=args[1])
+        body = { "openconfig-nat:config": {"access-list": args[2], "nat-pool": args[3]} }
+        l = len(args)
+        if l >= 5:
+            body["openconfig-nat:config"].update( {"nat-type": nat_type_map[args[4]] } )
+        if l == 6:
+            body["openconfig-nat:config"].update( {"twice-nat-id": args[5]} )
+        return api.patch(path, body)
+
+    # Remove NAT Binding
+    elif func == 'delete_openconfig_nat_nat_instances_instance_nat_acl_pool_binding_nat_acl_pool_binding_entry_config':
+        path = cc.Path('/restconf/data/openconfig-nat:nat/instances/instance={id}/nat-acl-pool-binding/nat-acl-pool-binding-entry={name}/config', id=args[0],name=args[1])
+        return api.delete(path)
+
+    # Remove all NAT Bindings
+    elif func == 'delete_openconfig_nat_nat_instances_instance_nat_acl_pool_binding':
+        path = cc.Path('/restconf/data/openconfig-nat:nat/instances/instance={id}/nat-acl-pool-binding', id=args[0])
+        return api.delete(path)
+
+    # Config NAT Zone
+    elif func == 'patch_openconfig_interfaces_ext_interfaces_interface_nat_zone_config_nat_zone':
+        path = cc.Path('/restconf/data/openconfig-interfaces:interfaces/interface={name}/openconfig-interfaces-ext:nat-zone/config/nat-zone', name=args[1])
+        body = { "openconfig-interfaces-ext:nat-zone": int(args[2]) }
+        return api.patch(path, body)
+
+    # Remove NAT Zone
+    elif func == 'delete_openconfig_interfaces_ext_interfaces_interface_nat_zone_config_nat_zone':
+        path = cc.Path('/restconf/data/openconfig-interfaces:interfaces/interface={name}/openconfig-interfaces-ext:nat-zone/config/nat-zone', name=args[1])
+        return api.delete(path)
+
+
+    ### GET
+    ## Get NAT Global Config
+    elif func == 'get_openconfig_nat_nat_instances_instance_config':
+        path = cc.Path('/restconf/data/openconfig-nat:nat/instances/instance={id}/config', id=args[0])
+        return api.get(path)
+
+    else:
+        return api.cli_not_implemented(func)
+
+def run(func, args):   
+
+    try:
+       args.insert(0,"0")  # NAT instance 0
+       response = invoke_api(func, args)    
+       if response.ok():
+           if response.content is not None:
+               # Get Command Output
+               api_response = response.content
+               show_cli_output(args[1], api_response)
+       else:
+           print response.error_message()
+
+        
+    except Exception as e:
+        print("Exception when calling OpenconfigNatApi->%s : %s\n" %(func, e))
```

Comment:
Let's not expose this to user. "%Error: Transaction failure." Or something similar.